examples/v1beta1/pipelineruns/pipelinerun.yaml

```diff
@@ -167,9 +168,11 @@ spec:
       runAsUser: 0
   - name: write-digest
     workingDir: $(workspaces.source.path)
-    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/imagedigestexporter:v0.11.1
+    image: ko://github.com/tektoncd/pipeline/cmd/imagedigestexporter
     # output of imagedigestexport [{"name":"image","digest":"sha256:eed29..660"}]
     command: ["/ko-app/imagedigestexporter"]
+    securityContext:
+      runAsUser: 0
```

Comment:
Because `kaniko` run as root, this has to run as root too, otherwise `permission denied` :sweat: